topic_monitor/setup.py

```diff
@@ -1,17 +1,27 @@
-from ament_python.data_files import get_data_files
-from ament_python.script_dir import install_scripts_to_libexec
+import sys
+
 from setuptools import find_packages
 from setuptools import setup
 
 package_name = 'topic_monitor'
-data_files = get_data_files(package_name)
-install_scripts_to_libexec(package_name)
+
+# install scripts into libexec path
+if 'develop' in sys.argv:
+    sys.argv[sys.argv.index('develop') + 1:sys.argv.index('develop') + 1] = \
```

Comment:
> I'm not sure that this will work.

Please try this PR together with the one on `ament_tools`. It installs the scripts into the libexec path as desired. Since that works I am pretty confident the Debian rules file is able to achieve the same behavior by invoking the same command.